probe_scraper/parsers/histograms.py

```diff
@@ -46,6 +46,10 @@ def extract_histogram_data(histogram, version):
             value = defaults[source_field]
         set_in_nested_dict(data, target_field, value)
 
+    # Only include labels if the histogram is categorical.
+    if histogram.kind() == "categorical":
+        set_in_nested_dict(data, "details/labels", histogram.labels())
```

Comment:
What will this value be for categorical histograms that don't have labels defined?

Reply:
There aren't any in the current histograms.json file, but I could guard against that and add an empty list in that case.